### ISSUE TITLE
[Request] Improve the Docs

### ISSUE THREAD

**Issue by katabame** [JamesWilko/Payday-2-BLT#97]
Hi BLT team.
You moved the BLT docs to GithubWiki from readthedocs, It's okay. but-
there is some of _Outdated_ breaking changes in BLT2. and page of those methods are not added yet.
Is there any methods instead of outdated one? or we can't use those anymore?

anyways, adding the pages is better I think.

**Comment by BangL** [comment]
i think wilko has more important stuff to do for now. fixing the blt for windows 7 for example.
until then you can PM me on steam if you need help updating stuff.

**Comment by ghost** [comment]
I agree that the docs could be improved. However, as said above, James would probably rather be working on more important things, such as fixing BLT2 for Windows 7 players.
